tensorboard/plugins/debugger_v2/tf_debugger_v2_plugin/views/alerts/alerts_component.css

```diff
@@ -13,4 +13,63 @@ See the License for the specific language governing permissions and
 limitations under the License.
 ==============================================================================*/
 
-/* TODO(cais): Flesh out the styling. */
+.alerts-breakdown-container {
+  font-size: 13px;
+  padding: 10px 10px 10px 50px;
+  position: relative;
+}
+
+.alerts-container {
+  font-family: 'Roboto', Arial, Helvetica, sans-serif;
+  vertical-align: middle;
+}
+
+.alert-type-count {
+  /* TODO(cais): Use colors specific to alert type. */
+  background-color: #e52592;
+  border-radius: 3px;
+  color: #fff;
+  display: inline-block;
+  padding: 3px;
+  position: absolute;
+  right: 20px;
+}
+
+.alert-type-name {
+  display: inline-block;
+}
+
+.debugging-title {
+  font-size: 18px;
+}
+
+.num-alerts-container {
+  font-weight: bold;
+  padding: 10px 10px 10px 30px;
+  position: relative;
+}
+
+.num-alerts-label {
+  display: inline-block;
+  font-size: 13px;
+}
+
+.num-alerts-value {
+  border-radius: 12px;
+  display: inline-block;
+  font-size: 13px;
+  font-weight: normal;
+  line-height: 24px;
+  position: absolute;
+  right: 20px;
+  text-align: center;
+  vertical-align: middle;
+  width: 24px;
+  -moz-border-radius: 12px;
```

Comment:
border-radius is pretty standard nowadays and you should rather use tools for these.